test/s3-s3.js

```diff
@@ -80,207 +80,225 @@ var
     };
   };
 
-describe('s3-s3', function(){
```

Comment:
I grouped the tests at a second level, since it was getting confused as-is.  The only real addition in here I'll comment on below